languages/csharp/sensitivity/field/constructors.tmt

```diff
@@ -0,0 +1,117 @@
+~extensions start~
+~[MACRO_Empty_string_Array]~ -> {"", ""}
+~[MACRO_Empty_string_Array]~ -> {"", "", ""}
+~[MACRO_Empty_string_Array]~ -> {"", "", "", ""}
+
+~[MACRO_Zero_Or_One]~ -> 0
+~[MACRO_Zero_Or_One]~ -> 1
+~extensions end~
```

Comment:
probably missed `~extensions import *~`

Reply:
Why would that be needed?

Reply:
File contains some holes, ~[EXPR_bool]~ for example